daml-assistant/daml-helper/src/DA/Daml/Helper/Run.hs

```diff
@@ -732,13 +733,27 @@ runDeploy flags = do
     putStrLn "Deploy succeeded."
     exitSuccess
 
+newtype JsonFlag = JsonFlag { unJsonFlag :: Bool }
+
 -- | Fetch list of parties from ledger.
-runLedgerListParties :: HostAndPortFlags -> IO ()
-runLedgerListParties flags = do
+runLedgerListParties :: HostAndPortFlags -> JsonFlag -> IO ()
+runLedgerListParties flags (JsonFlag json) = do
     hp <- getHostAndPortDefaults flags
-    putStrLn $ "Listing parties at " <> show hp
+    unless json . putStrLn $ "Listing parties at " <> show hp
     xs <- Ledger.listParties hp
-    if null xs then putStrLn "no parties are known" else mapM_ print xs
+    if json then do
+        putStr . T.unpack . TL.toStrict . encodeToLazyText . toJSON $
```

Comment:
A trailing newline still is nice even when outputting JSON imho.

Reply:
```suggestion
        Text.Lazy.IO.putStrLn . encodeToLazyText . toJSON $
```

Reply:
👍  Thanks!